### DISCUSSION TITLE
✨  Meet the new GitBook ✨

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#410] [Announcements]
Hello everyone! Today we have some super exciting news for you all — we’ve made some major updates to GitBook! ✨ 

<img width="1024" alt="14" src="https://github.com/GitbookIO/community/assets/42930383/d5bad451-9caa-40fb-a6a9-7da2c0b4d623">

## Powerful new features
We’ve launched a suite of new features in the GitBook app, making it easier than ever to document and maintain your team’s technical knowledge. You’ll probably spot some of them in the new sidebar as soon you log in!

## More ways to use your favorite tools
Our new **Slack** [beta] and **VS Code** [alpha] integrations let you hook GitBook up to the tools you use every day. So you can create docs as you work in VS Code — or find useful info from your knowledge base in seconds, right in your IDE.
 
And with our Slack integration, GitBook can capture complex information directly from a conversation thread, then extract the valuable knowledge and turn it into documentation. Or help you find answers from your knowledge base, right in a chat.

## Major UI updates
That’s not all! We’ve also redesigned the entire app — including new colors, fonts, icons, and illustrations! Make sure you check out how the app looks in both light  and dark mode to see the new UI in full effect!

## Updated branding
Finally, we can’t give an update without highlighting our new brand. With so many changes to the platform, it was only right to compliment it with an entirely new look across the board.

## Share your thoughts on Product Hunt!
If you have any feedback, we’d love to hear it. We're live on ProductHunt where you can explore more and share your thoughts!

[Support our launch](https://www.producthunt.com/posts/meet-the-all-new-gitbook)

For now, enjoy using the new features — we can’t wait to hear what you think!

_Addison + the GitBook team_

**Comment by JaniceManwiller** [comment]
Please restore the **Create change request** option to the **Change Requests** panel. It is very confusing to not have it there - and to have go back up to click the **Edit** option after checking the current change requests. **Edit** also does not intuitively to me say to create a change request - I would assume that would only be used for direct editing without the change request process.

**Reply by JaniceManwiller** [reply]
Actually, it looks like the option is there if there are no change requests at all, but disappears as soon as there are any existing change requests. Would always like the create change request to always be available on the panel.

**Reply by addisonschultz** [reply]
Hi @JaniceManwiller ! We did remove it in the cases there are existing Change Requests in the space. You can still make a new CR by clicking the Edit button as you've mentioned! We're going to take another look at the UX of this flow, but hopefully in the meantime the edit button will work

**Comment by nikitin-pro** [comment]
Hello, I actually like restyling, except checkboxes. In editor they are looking fine, but on published pages one can barely distinguish between checked and unchecked items (try it on mobile in sunny day while walking).
![image](https://github.com/GitbookIO/community/assets/95876501/a53dae60-9bf7-4a17-bf24-25d2f492d06e)

**Comment by JeongJun-Lee** [comment]
The table of contents on the left in the published pages has disappeared in the new UI. It is difficult for users to quickly navigate between each page. Could you give me it back?

**Reply by addisonschultz** [reply]
Hi @JeongJun-Lee - It should still be there! Check the side for an icon that will allow you to toggle the view!

<img width="635" alt="Screenshot 2024-04-17 at 14 30 57" src="https://github.com/GitbookIO/community/assets/42930383/ff61ddc5-e371-484b-b82f-2ccc5b3e92c7">

**Reply by JeongJun-Lee** [reply]
@addisonschultz Thanks, but I mean the published pages doesn't have the menu.

**Reply by addisonschultz** [reply]
Can you share your published space?

**Reply by JeongJun-Lee** [reply]
https://ufe.gitbook.io/entry_basic

**Reply by addisonschultz** [reply]
It looks like the menu is still there for me - can you show me what you see? Is your screen zoomed in?

<img width="1624" alt="Screenshot 2024-04-19 at 11 01 37" src="https://github.com/GitbookIO/community/assets/42930383/e099f9ae-2357-4365-8260-5f193dba467c">

**Reply by JeongJun-Lee** [reply]
<img width="1673" alt="image" src="https://github.com/GitbookIO/community/assets/11474775/3a6f3d4b-b959-4808-ab08-492143c41f46">

Oh it's only problem on my computer, right? I didn't zoom it. Even I cleared the cache in the Chrome browser and restarted the Chrome as well, but the result is same on my computer. The table of contents on the left in the published pages has not shown to me.

**Reply by addisonschultz** [reply]
Can you try to zoom out? And also try a different browser?

**Reply by roboticsware** [reply]
My chrome browser is not woking even the zoom out, but my Safari browser works nomally.

**Comment by Exifers** [comment]
Is there a way to get the old ui (for the published documentation)? I liked the bluish look and feel.
